nbgitpuller/handlers.py

```diff
@@ -1,17 +1,27 @@
 from tornado import gen, web, locks
 import traceback
 import urllib.parse
-
 from notebook.base.handlers import IPythonHandler
 import threading
 import json
 import os
```

Comment:
```suggestion
import os
from functools import partial
```

To complement a code suggestion below